modules/installation-configuration-parameters.adoc

```diff
@@ -627,6 +627,12 @@ endif::openshift-origin[]
 |Specify one or more repositories that may also contain the same images.
 |Array of strings
 
+ifdef::aws[]
+|`platform.aws.lbType`
+|Required to set the NLB load balancer type in AWS. Valid values are `Classic` or `NLB`. The default is Classic if no value is provided. The installation program sets the value provided here in the ingress cluster config object. If you do not specify a load balancer type for other ingress controllers, they will use the configuration set in this parameter.
+|String 'Classic' or 'NLB'.
```

Comment:
```suggestion
|`Classic` or `NLB`. The default value is `Classic`.
```
I think this is the design pattern used in the install docs. @bscott-rh Would you please confirm?

Also, backticks, not single quotes, for literals.